pkg/doc/verifiable/example_credential_test.go

```diff
@@ -342,11 +342,11 @@ func ExampleCredential_AddLinkedDataProof() {
 	}
 
 	err = vc.AddLinkedDataProof(&verifiable.LinkedDataProofContext{
-		Created:       &issued,
-		Creator:       "Example University",
-		SignatureType: "Ed25519Signature2018",
-		Suite:         ed25519signature2018.New(),
-		PrivateKey:    privIssuerKey,
+		Created:                 &issued,
+		SignatureType:           "Ed25519Signature2018",
```

Comment:
better to create a constant for `"Ed25519Signature2018"`

Reply:
I think it's OK to use the constant as is in the test.